css/quiz.css

```diff
@@ -0,0 +1,84 @@
+* {
```

Comment:
Ryan,

Pushed an update to my Quiz.

Thanks for the help, I had the tab thing totally wrong, not just the HTML
section. I thought he had told us to check "Tab Width 2" & "Indent using
Spaces". That makes no sense having both selected does it? I was thinking
that the indent using spaces was the option that removed extra accidental
spaces after tags, not sure what that was, maybe a package we installed?

Anyway, let me know how it looks now.

Thanks,
Simon

On Mon, Apr 11, 2016 at 2:51 PM, Ryan Tanay <notifications@github.com>
wrote:

> In css/quiz.css
> <https://github.com/nss-day-cohort-13/static-web-html-css-quiz-SimonHerrera/pull/1#discussion_r59268991>
> :
>
> > @@ -0,0 +1,84 @@
> > +* {
>
> Your indentation has mixed tabs and spaces. While the holy war on
> indentation can go either way for the tabs vs. spaces camp, the indentation
> should be consistent throughout the document/project. Pick the indentation
> style you'd prefer and apply it throughout the document.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nss-day-cohort-13/static-web-html-css-quiz-SimonHerrera/pull/1/files/10f8c73c89558af67d0d9e66fa29c5b8df296085#r59268991>
>